contracts/src/rollup/RollupUserLogic.sol

```diff
@@ -340,8 +340,9 @@ abstract contract AbsRollupUserLogic is
             globalStates,
             numBlocks,
             wasmModuleRoots,
-            commonEndTime - proposedTimes[0],
-            commonEndTime - proposedTimes[1]
+            // convert from block counts to real second based timestamps
+            (commonEndTime - proposedTimes[0]) * 12,
+            (commonEndTime - proposedTimes[1]) * 12
```

Comment:
maybe we can rename `commonEndTime` to `commonEndBlock` and etc. to avoid confusion, also can make `12` a constant e.g. `ETH_POS_BLOCK_TIME`